<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>into</class>
 <widget class="QWidget" name="into">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход/Регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Безопасное дорожное движение</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="status">
     <property name="text">
      <string>Введите ваши данные:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="login">
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password">
       <property name="maxLength">
        <number>128</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="reg_input_f">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Снова:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="password_again">
        <property name="maxLength">
         <number>128</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="name_first">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="name_second">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="log_buttons_f">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="log_b">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(53, 52, 77);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Вход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reg_open_b">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(53, 52, 77);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="reg_buttons_f">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="reg_b">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(53, 52, 77);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reg_close">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(53, 52, 77);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reg_open_b</sender>
   <signal>clicked()</signal>
   <receiver>reg_input_f</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reg_open_b</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reg_open_b</sender>
   <signal>clicked()</signal>
   <receiver>password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reg_open_b</sender>
   <signal>clicked()</signal>
   <receiver>reg_buttons_f</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reg_open_b</sender>
   <signal>clicked()</signal>
   <receiver>log_buttons_f</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
